setup.py

```diff
@@ -41,12 +52,13 @@
 AUTHOR = "Matthias Kümmerer"
 EMAIL = "matthias.kuemmerer@bethgelab.org"
 URL = "https://github.com/matthias-k/cyipopt"
-DEPENDENCIES = ["numpy>=1.15",
-                "cython>=0.26",
-                "future>=0.15",
-                "setuptools>=39.0",
-                "six>=1.11"
-                ]
+INSTALL_REQUIRES = [
+    "numpy>=1.15",
+    "cython>=0.26",
+    "future>=0.15",
+    "setuptools>=39.0",
+    "six>=1.11",
```

Comment:
It is likely that cython and setuptools are not needed to run the library. We'd need to double check.

Reply:
I think Cython could be removed here, but not necessarily numpy. Shouldn't setuptools be a build dep then too? I don't see it used anywhere but `setup.py`.

Reply:
true ... I did not want to meddle too much with the build, but certainly possible that cython/numpy can be removed under install_requires (here I did not change those dependencies at all).

setuptools should technically be a build dep too, however I guess that would only work via pyproject.toml or setup.cfg, as setup.py is completely based on setuptools. afaik, it is usually pre-installed on python distros.

Reply:
Understood. The file requires an import of setuptools itself, so it has to be pre-installed. We can leave this as is.